packages/core/src/utils/pipeHandler.ts

```diff
@@ -5,6 +5,29 @@ import { PlateEditor } from '../types/PlateEditor';
 import { DOMHandlers, HandlerReturnType } from '../types/plugin/DOMHandlers';
 import { TEditableProps } from '../types/slate-react/TEditableProps';
 
+function convertDomEventToSyntheticEvent(domEvent: Event) {
```

Comment:
Can you use arrow function syntax here and export the function? Also, do the types work if we explicitly set the return type here to `React.SyntheticEvent<unknown, unknown>`?